packages/api-client/README.md

```diff
@@ -214,6 +215,23 @@ fromCatalog()
     .lang('fr'); // Force a language
 ```
 
+# Versioning
+
+The client must follow [Opendatasoft's Explore API](https://help.opendatasoft.com/apis/ods-explore-v2/) evolution.
+However, versioning is handled differently by the two projects. This package follows [Semantic Versioning](https://semver.org/) but not the Opendatasoft's Explore API (between the 2.0 and 2.1 there are some [breaking changes](https://help.opendatasoft.com/apis/ods-explore-v2/explore_v2.1.html#section/Introduction/v2.1-Changelog)).
+
+The table below shows how the api client follows Opendatasoft's Explore API evolution according to the type of change that may occur.
+
+| Opendatasoft's Explore API version | Api client version  | Comment |
+|---|----|----|
+| 2.0  | 0.x.x   | Api client was in beta                                                                   |
+| 2.1  | 21.0.0  | 2.0 ⇒ 2.1 - Breaking change in Explore API - So major version upgrade for the api client |
+| 3.0  | 30.0.0  | 2.0 ⇒ 3.0 - Paradigm change in Explore API - So major version upgrade for the api client |
+| 3.0  | 30.1.0  | No changes in Explore API but a new feature in the api client                            |
+| 3.0  | 30.1.1  | No changes in Explore API but a fix in the api client                                    |
+
+*When the version of Opendatasoft's Explore API will upgrade from 2.x to 3.0, a version 22.0.0 of the api client could be created, but to easily remember the correspondence between the two packages, the api client version will be 30.0.0.*
```

Comment:
I think we should remove this part for the sake of clarity. This was originally written to explain the logic in a presentation, but here keeping it factual and a reference explanation seems enough and non-ambiguous. (that's an opinion at first read, I may be wrong)